Doc/library/subprocess.rst

```diff
@@ -654,6 +655,17 @@ functions.
    .. versionadded:: 3.10
       The ``pipesize`` parameter was added.
 
+   *read_callback* can be used to replace the default internal handler for
+   reading the :data:`PIPE` used for *stdout* or *stderr* of the child process.
+   Replacements must accept the same parameters, which are ``self``,
+   ``is_stdout`` (bool), the ``buffer`` (bytes) to accumulate data into,
+   and the ``data`` (bytes) just read from the pipe.
+
+   *tee* when set, indicates that the pipe reader callback should not only
```

Comment:
This should be described without mentioning a reader callback at all. That is an implementation detail. Rather than a parameter for this, perhaps providing a predefined read callback that implements tee would be better.  combined with splitting the callback into two, this would allow people to tee one or the other easily.

Reply:
I've removed mention of the reader callback.